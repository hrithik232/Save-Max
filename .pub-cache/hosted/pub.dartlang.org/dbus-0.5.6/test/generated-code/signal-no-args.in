<node>
  <interface name="com.example.Test">
    <signal name="Event"/>
  </interface>
</node>
