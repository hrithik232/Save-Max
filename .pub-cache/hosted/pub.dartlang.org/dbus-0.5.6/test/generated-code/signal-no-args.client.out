import 'package:dbus/dbus.dart';

/// Signal data for com.example.Test.Event.
class ComExampleTestEvent extends DBusSignal {
  ComExampleTestEvent(DBusSignal signal) : super(signal.sender, signal.path, signal.interface, signal.name, signal.values);
}

class ComExampleTest extends DBusRemoteObject {
  /// Stream of com.example.Test.Event signals.
  late final Stream<ComExampleTestEvent> event;

  ComExampleTest(DBusClient client, String destination, DBusObjectPath path) : super(client, destination, path) {
    event = DBusRemoteObjectSignalStream(this, 'com.example.Test', 'Event', signature: DBusSignature('')).map((signal) => ComExampleTestEvent(signal));
  }
}
